pkg/users/service.go

```diff
@@ -0,0 +1,40 @@
+package users
+
+import (
+	"fmt"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/logging"
+)
+
+// Service specifies an API to the domain service.
+type Service interface {
+	// Authenticate validates the given token. If it is valid,
+	// nil error value is returned. If the token in invalid,
+	// a non-nil error value is returned.
+	Authenticate(token string) error
```

Comment:
Don't you think this method is self-explained? I think the comment can be removed.

Reply:
Done :smile: